### PR TITLE
Document Mergify v2

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: doc/source/configuration.rst [hunk 1]
```diff
@@ -1,213 +1,139 @@
...
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic merge for Dependabot pull requests on master
+        conditions:
+          - author=dependabot[bot]
+          - status-success=continuous-integration/travis-ci
+          - base=master
+        merge:
```
Comment (sileht) [inline doc/source/configuration.rst]: `actions:` is missing

File: doc/source/configuration.rst [hunk 2]
```diff
@@ -1,213 +1,139 @@
...
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic merge for master when reviewed and CI passes
+        conditions:
+          - status-success=continuous-integration/travis-ci
+          - "#review-approved-by>=2"
+          - base=master
+        merge:
```
Comment (sileht) [inline doc/source/configuration.rst]: same here

File: doc/source/configuration.rst [hunk 3]
```diff
@@ -1,213 +1,139 @@
...
+          method: merge
+      - name: automatic merge for stable branches
+        conditions:
+          - status-success=continuous-integration/travis-ci
+          - "#review-approved-by>=1"
+          - base~=^stable/
+        merge:
```
Comment (sileht) [inline doc/source/configuration.rst]: same

File: doc/source/configuration.rst [hunk 4]
```diff
@@ -1,213 +1,139 @@
...
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic merge for master when reviewed and CI passes
+        conditions:
+          - status-success=continuous-integration/travis-ci
+          - "#review-approved-by>=2"
+          - base=master
+          - label=ready-to-merge
+        merge:
```
Comment (sileht) [inline doc/source/configuration.rst]: same

File: doc/source/configuration.rst [hunk 5]
```diff
@@ -1,213 +1,139 @@
...
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic merge for master when reviewed and CI passes
+        conditions:
+          - status-success=continuous-integration/travis-ci
+          - "#review-approved-by>=2"
+          - base=master
+          - label!=work-in-progress
+        merge:
```
Comment (sileht) [inline doc/source/configuration.rst]: same

File: doc/source/getting-started.rst [hunk 6]
```diff
@@ -26,103 +29,67 @@ Here's how to create a file with a minimal content to enable Mergify:
...
+.. code-block:: yaml
 
-You can define a per branch rule using the ``branches`` settings. To match
-multiple branches at once, you can use a regular expression:
+    pull_request_rules:
+      - name: automatic merge on CI success and review
+        conditions:
+        - status-success=continuous-integration/travis-ci
+        - "#review-approved-by>=2"
+        merge:
```
Comment (sileht) [inline doc/source/getting-started.rst]: same

**Comment by sileht** [comment]
All `review-approved-by` must be changed to`approved-reviews-by`.

**Comment by jd** [comment]
Pull-request updated, HEAD is now d96c153861a64131ae51fa064bd9cc09cd9ca18f

**Comment by jd** [comment]
Added label action

**Comment by jd** [comment]
Add doc for dismiss_reviews

**Comment by jd** [comment]
Add close action

**Comment by jd** [comment]
Add delete_head_branch

**Comment by jd** [comment]
Pull-request updated, HEAD is now ec2d3467f978b24330a829de08a67b123603b7c4

**Comment by jd** [comment]
Move examples to a page

**Comment by jd** [comment]
Add smart strict mode

**Pull request merged**